### PR TITLE
DDF-2943 Fixes Spock tests running

### PR DESCRIPTION
**Pull request by rzwiefel**
#### What does this PR do?
Previously spock unit tests would not be ran unless they were suffixed with *Test. 
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
anyone

#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).
[Test](https://github.com/orgs/codice/teams/test)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@coyotesqrl 
@lessarderic

#### How should this be tested? (List steps with links to updated documentation)
Ensure spock tests not ending in `Test` are ran and included in coverage

#### What are the relevant tickets?
https://codice.atlassian.net/browse/DDF-2943

### CODE REVIEW

**Comment by rzwiefel** [comment]
build now

**Comment by vinamartin** [comment]
Are the tests that weren't running passing?
That's a miracle! 🎉

File: pom.xml [hunk 1]
```diff
@@ -276,6 +276,12 @@
                     <version>2.8.1</version>
                     <configuration>
                         <argLine>${argLine} -Djava.awt.headless=true -noverify</argLine>
+                        <includes>
+                            <include>**/*.java</include>
```
Comment (coyotesqrl) [inline pom.xml]: Spoke with @rzwiefel about this. It's confounding. Not clear how it doesn't now skip all Spock tests; not clear how it _fails_ if he includes `**/*.groovy` as an include pattern.

@lessarderic do you have some insight? It's not like the groovyc compiler is generating java files as an interim step...

Comment (lessarderic) [inline pom.xml]: Looks like this is a known issues: http://stackoverflow.com/a/34308282.

Could we do this instead? I tried and it seems to work:
```
<includes>
  <include>**/*Test.java</include>
  <include>**/Test*.java</include>
  <include>**/*TestCase.java</include>
  <include>**/*Spec.class</include>
```

Don't know if that's the plan but we should update all the other projects as well (e.g., Alliance).

Comment (rzwiefel) [inline pom.xml]: Hmmm I thought that we had test classes without Test on them at all (which then failed coverages checks). Will try this though and let you know :+1: Good find.

Comment (lessarderic) [inline pom.xml]: Shouldn't be the case. The default inclusion list only includes the first three lines I put in my comment.

**Comment by rzwiefel** [comment]
build now

File: pom.xml (hunk 1 above)
Comment (rzwiefel) [inline pom.xml]: Updated to the above list.

**Pull request merged**